TSOClient/tso.common/utils/PathCaseTools.cs

```diff
@@ -12,7 +12,8 @@ public static class PathCaseTools
         public static string Insensitive(string file)
         {
             var dir = Directory.GetFiles(Path.GetDirectoryName(file));
-            return dir.FirstOrDefault(x => x.ToLowerInvariant().Replace('\\','/') == file.ToLowerInvariant().Replace('\\', '/'));
+            var name = Path.GetFileName(file).ToLowerInvariant();
```

Comment:
Why is this changed? This will break linux and osx.

Reply:
That line crashed my game on loading sounds. I believe it was due to my InstallPath variable in the registry having an extra slash at the end. I thought that other users might have a similar problem if their registry entry was the same way.

Reply:
Also it could actually break a ridiculous number of things since it will compare on filename and not the entire path. You should find out where the path is being joined, something is probably using + instead of Path.Join, or you can sanitize the path to not have a trailing slash when it is set on the content object. But not this.